getting-started/dcr-guide.rst

```diff
@@ -0,0 +1,102 @@
+######################################
+Generating DCR Coadds with LSST ComCam
+######################################
+
+A walkthrough for building differential chromatic refraction (DCR) coadds by running the
+`dcrAssembleCoaddTask` on an example set of ComCam image data. Prior to generating DCR coadds, you must
+have image templates.
+
+.. note::
+
+   This guide assumes the user has access to a shared Butler repository containing data from LSST ComCam via
+   the `US Data Facility (USDF) <https://developer.lsst.io/usdf/storage.html>`__. This guide further assumes
+   the user has a recently-built version of ``lsst.distrib`` from the `LSST Science Pipelines
+   <https://developer.lsst.io/usdf/stack.html>`__ (circa ``w_2022_30`` or later).
+
+Generating AP Templates
+=======================
+For detailed instructions on generating AP Templates, please see the `Building good seeing templates
+<https://pipelines.lsst.io/getting-started/dc2-guide.html>`__ guide for details.
+
+
+Assembling DCR Coadds
+=====================
+To generate DCR coadds, you must use the `dcrAssembleCoaddTask` found in the `drp_tasks` package. This will
+require image templates. Note: You will need to register your DCR subfilters prior to creating DCR coadds by
+using the following code:
+
+.. code-block:: python
+    butler register-dcr-subfilters /repo/main 3 'g'
+
+
+Creating Your DCR Assemble Coadd Pipeline
+-----------------------------------------
+The first step in generating DCR coadds is to make a pipeline that runs the `dcrAssembleCoaddTask` from the
+LSST 'drp_tasks' package. A sample `dcrAssembleCoadd` pipeline is shown below.
+
+.. code-block:: python
+
+    :name: dcrAssembleCoadd.yaml
+
+    description: The DCR assemble coadd pipeline for LSSTComCam
+
+    parameters:
+      coaddName: dcr
+    tasks:
+      DcrAssembleCoaddTask:
+        class: lsst.drp.tasks.dcr_assemble_coadd.DcrAssembleCoaddTask
+        config:
+          effectiveWavelength: 478.5
+          bandwidth: 147.0
+
+To visualize your pipeline, you may wish to use `pipetask build` e.g.,
+
+.. code-block:: shell
+    pipetask build -p $AP_PIPE_DIR/pipelines/LSSTComCam/dcrAssembleCoadd.yaml --pipeline-dot dcrCoadd.dot
+    dot dcrCoadd.dot -Tpng > dcrCoadd.png
+
+
+Run `dcrAssembleCoadd` Pipeline
```

Comment:
Personal preference, but I'd follow the same pattern you used for the next subsection:
```suggestion
Assembling DCR Coadds with ``pipetask run``
```

Alternatively:
```suggestion
Run ``dcrAssembleCoadd`` Pipeline
```